src/stories/components/c-modal.stories.js

```diff
@@ -22,61 +22,73 @@ export default {
     size: 600,
     spacing: 500,
     stickyFooter: false,
-    content: faker.lorem.paragraphs(),
+    content: faker.lorem.paragraph(),
   },
 };
+const template = `
```

Comment:
todo: please use a different name, since we already have `Template`.

Reply:
`template` is the default property name and also reflects the `<template>` part of the Vue SFC.
https://storybook.js.org/docs/vue/writing-stories/introduction#stories-for-two-or-more-components

I would rather go ahead and use a different name for the `const Template` on line 55.